packages/office-ui-fabric-react/src/components/ColorPicker/ColorSlider/ColorSlider.base.tsx

```diff
@@ -1,57 +1,40 @@
 import * as React from 'react';
-import { BaseComponent, classNamesFunction } from '../../../Utilities';
+import { classNamesFunction, initializeComponentRef, EventGroup, warnDeprecations } from '../../../Utilities';
 import { IColorSliderProps, IColorSliderStyleProps, IColorSliderStyles } from './ColorSlider.types';
 
 const getClassNames = classNamesFunction<IColorSliderStyleProps, IColorSliderStyles>();
 
-/**
- * {@docCategory ColorPicker}
- */
-export interface IColorSliderProps {
-  componentRef?: () => void;
-  minValue?: number;
-  maxValue?: number;
-  value?: number;
-  thumbColor?: string;
-  overlayStyle?: any;
-  onChange?: (event: React.MouseEvent<HTMLElement>, newValue?: number) => void;
-  onChanged?: (newValue: number) => void;
-
-  className?: string;
-  style?: any;
-}
-
 export interface IColorSliderState {
-  isAdjusting?: boolean;
-  origin?: { x: number; originalValue: number };
   currentValue?: number;
 }
 
 /**
  * {@docCategory ColorPicker}
  */
-export class ColorSliderBase extends BaseComponent<IColorSliderProps, IColorSliderState> {
+export class ColorSliderBase extends React.Component<IColorSliderProps, IColorSliderState> {
   public static defaultProps = {
     minValue: 0,
     maxValue: 100,
     thumbColor: 'inherit',
     value: 0
   };
 
+  private _events: EventGroup;
   private _root = React.createRef<HTMLDivElement>();
 
   constructor(props: IColorSliderProps) {
     super(props);
 
-    this._warnDeprecations({
+    initializeComponentRef(this);
```

Comment:
Same about BaseComponent

Reply:
This one already has BaseComponent removed